<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>721</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="generate">
     <attribute name="title">
      <string>生成图文件</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>结点数(非追加模式)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>边数</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>101</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="NormalModeButton">
         <property name="text">
          <string>正常模式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="OneConnectModeButton">
         <property name="text">
          <string>单连通块</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="MultyConnectButton">
         <property name="text">
          <string>多联通块</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="AddModeButton">
         <property name="text">
          <string>追加模式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="NoSelfLoopButton">
         <property name="text">
          <string>去除自环</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="NoMultyEdgeButton">
         <property name="text">
          <string>去除重边</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="GenerateButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>330</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>生成图文件</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="InputLocation">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SelectPathButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>选择位置</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>130</y>
        <width>451</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ModeSelect">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>89</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>模式选项</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="VerNumInput">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="EdgeNumInput">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="BlockNumInput">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>连通块个数(多联通块)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ClearTextBrowserButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>330</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>清空日志</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>检测图文件</string>
     </attribute>
     <widget class="QTextBrowser" name="LogBrowser">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>611</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>结点数据类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="IntFormatButton">
         <property name="text">
          <string>整形</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="StringFormatButton">
         <property name="text">
          <string>字符串</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="ClearLogButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>250</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>清空日志</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CheckMinDistButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>390</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>最短路距离</string>
      </property>
     </widget>
     <widget class="QPushButton" name="InputSubGraphButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>340</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>导入子图</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CheckButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>340</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>检查</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CheckUnicomButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>390</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>检查连通性</string>
      </property>
     </widget>
     <widget class="QPushButton" name="InputGraphButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>300</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>导入原图</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NewCheckerButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>300</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>生成检查器</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>370</y>
        <width>45</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>结点2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>370</y>
        <width>45</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>结点1</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="InputGraph">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Ver1Input">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>390</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Ver2Input">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>390</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="InputSubGraph">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>340</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
